src/components/ApplyFiltersModal/DefaultFilter.tsx

```diff
@@ -3,32 +3,53 @@ import Badge from "@leafygreen-ui/badge";
 import Checkbox from "@leafygreen-ui/checkbox";
 import Tooltip from "@leafygreen-ui/tooltip";
 import Icon from "components/Icon";
+import { CaseSensitivity, MatchType } from "constants/enums";
 import { size } from "constants/tokens";
 import { ParsleyFilter } from "gql/generated/types";
 import { Filter } from "types/logs";
 
 interface DefaultFilterProps {
+  addFilter: (filterToAdd: Filter) => void;
+  removeFilter: (filterToRemove: string) => void;
   activeFilters: Filter[];
+  selectedFilters: Filter[];
   filter: ParsleyFilter;
 }
 
 const DefaultFilter: React.FC<DefaultFilterProps> = ({
+  addFilter,
+  removeFilter,
   activeFilters,
+  selectedFilters,
   filter,
 }) => {
-  const alreadyActive = !!activeFilters.find(
-    (f) => f.name === filter.expression
-  );
+  const active = !!activeFilters.find((f) => f.name === filter.expression);
+  const selected = !!selectedFilters.find((f) => f.name === filter.expression);
```

Comment:
It seems odd that these values are calculated within the component instead of being passed in especially since they aren't being used for anything else.
Since this component is essentially a checkbox I don't think it should be responsible for determining its own state.

Reply:
+1